Sources/TokamakDemo/TokamakDemo.swift

```diff
@@ -135,7 +146,7 @@ struct TokamakDemoView: View {
         title: "Demos"
       )
       #if os(WASI)
-      return AnyView(list)
+      return AnyView(list.listStyle(SidebarListStyle()))
```

Comment:
You should be able to merge this in like I did in the `sidebarDemo`

Reply:
I am not sure what you want me to return here. In the change you made, you returned a NavigationItem. In this case, we just are returning a boring old list if the SidebarListStyle is not available.

Reply:
```swift
if #available(iOS 14.0, *) {
  return AnyView(list.listStyle(SidebarListStyle()))
} else {
  return AnyView(list)
}
```

Reply:
Isn't that exactly what is on lines 150-156 here?

Reply:
Yep! The `if os(WASI)` part shouldn’t be necessary because the `*` in the `#available` includes the WASI platform.

Reply:
Got it.